### PR TITLE
start august newsletter

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2023-07-25-news-august-2023/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### Open Call for rOpenSci Champions Program Applications! 
+Still open: [our call](/blog/2023/07/14/call-champions-program/) for the second cohort of Champions and Mentors for the rOpenSci Champions Program 🎉 ! 
+
+This 12-month-long program will continue to support our goal of identifying, recognizing, and rewarding passionate community members who help the community grow and improve.  The activities include cohort-based training, project development, and personal mentorship.
```
Comment (yabellini) [inline content/blog/2023-07-25-news-august-2023/index.Rmd:46]:
```suggestion
This 12-month-long program will continue to support our goal of identifying, recognizing, and rewarding passionate community members who help the community grow and improve.  The activities include [cohort-based training](/blog/2023/08/04/champions-program-training/), project development, and personal mentorship.
```

File: content/blog/2023-07-25-news-august-2023/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+
+This program focuses on people who belong to groups that are historically and systematically excluded from the open software and research software communities and who are interested in contributing to rOpenSci and the broader open source and research software communities.
+
+You can apply to be a _champion_ or a _mentor_ until **September 4th**.
```
Comment (yabellini) [inline content/blog/2023-07-25-news-august-2023/index.Rmd:50]:
```suggestion
You can apply to be a [champion](https://airtable.com/shrAsYlSXU0coJ5Ld) or a [mentor](https://airtable.com/shrlrLdQbb4wphosK) until **September 4th**.
```

File: content/blog/2023-07-25-news-august-2023/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+
```
Comment (yabellini) [inline content/blog/2023-07-25-news-august-2023/index.Rmd:51]:
```suggestion
Want to hear more from the protagonists?

In these videos (in [English](https://vimeo.com/848537831) and [Spanish](https://vimeo.com/854320485)) our champions and mentors tell us what it's like to be part of the program. 
```

File: content/blog/2023-07-25-news-august-2023/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,474 @@
...
+More information on eligibility, timeline, curriculum, application process, etc. is [on the program webpage](/champions/). 
+
+### Com
```
Comment (yabellini) [inline content/blog/2023-07-25-news-august-2023/index.Rmd:54]:
```suggestion
### Our first community call in Spanish!

As global movements, Open Source and Open Science face language-based exclusion as most resources are in English. This affects scientists and research software engineers working in R, particularly those who don’t have English as their first language.

rOpenSci multilingual efforts aim to lower access barriers, democratize quality resources, and increase the possibilities of contributing to open software and science. We successfully piloted our Spanish-language peer review and the localization to Spanish of our comprehensive guide to software development, with Portuguese translation underway.

[Maëlle Salmon](/author/ma%C3%ABlle-salmon/), [Paola Corrales](/author/paola-corrales/), and [Elio Campitelli](/author/elio-campitelli/),  will share the rOpenSci Multilingual project details on this call. Maëlle will present the R packages that allow us to have our content in several languages. Then Elio and Paola will share the translation workflow and show the Translation Guide written to document the process.

- **Multilingual Publishing**:  Tuesday, 21 November 2023 12:00 UTC. [More information](/commcalls/nov2023-multilingual/)

- **Proyecto Multilingüe**: Thursday, 23 November 2023 12:00 UTC. [More information](/commcalls/nov2023-multilenguaje/)


```

**Review by yabellini — COMMENTED** [review]
Adding champions program and community calls.

**Comment by yabellini** [comment]
The eph package is listed two times on the newsletter.  It is that correct?

File: content/blog/2023-07-25-news-august-2023/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,489 @@
...
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- Tuesday, September 5th, 09:00 Australia Western (01:00 UTC), [" AI in/for R"](/events/coworking-2023-09/) *Hosted by [Joel Nitta](/author/joel-nitta/) and [Steffi LaZerte](/author/steffi-lazerte/)*
+    - Explore how R can inform/perform AI as well as how AI can inform/perform R.
+    - Cowork independently on work related to R.
+- **Note** October coworking is cancelled (see you in November!)
```
Comment (steffilazerte) [inline content/blog/2023-07-25-news-august-2023/index.Rmd:92]:
```suggestion
- ~~**Note** October coworking is cancelled (see you in November!)~~ **Nope, it's back on!** Stay tuned for coworking Tuesday, October 3rd, 14h00 Europe Central (12:00 UTC), Theme TBA
```

File: content/blog/2023-07-25-news-august-2023/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,489 @@
...
+- **[rvertnet](https://cran.r-project.org/web/packages/rvertnet/index.html)**, Retrieve, map and summarize data from the VertNet.org archives (<https://vertnet.org/>). Functions allow searching by many parameters, including taxonomic names, places, and dates. In addition, there is an interface for conducting spatially delimited searches, and another for requesting large datasets via email. [Issue for volunteering](https://github.com/ropensci-archive/rvertnet/issues/71).
+
+- **[natserv](https://cran.r-project.org/web/packages/natserv/index.html)**. Interface to NatureServe (<https://www.natureserve.org/>). Includes methods to get data, image metadata, search taxonomic names, and make maps. [Issue for volunteering](https://github.com/ropensci-archive/natserv/issues/29).
+
+### Call for comaintainers
+
+Refer to our somewhat [recent blog post](/blog/2022/10/17/maintain-or-co-maintain-an-ropensci-package/#packages-looking-for-co-maintainers) to identify other packages where help is especially wished for!
+See also our [help wanted page](/help-wanted/) -- before opening a PR, we recommend asking in the issue whether help is still needed.
+
+## Package development corner
```
Comment (steffilazerte) [inline content/blog/2023-07-25-news-august-2023/index.Rmd:459]: So cool! I'm terrible at keeping up on things, and love that you do all the hard work for me ;)

File: content/blog/2023-07-25-news-august-2023/index.md [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+
+
+
+### Other topics
+
+* [Teaching targets with Penguins](/blog/2023/07/20/teaching-targets-with-penguins) by Athanasia Mo Mowinckel, and Joel Nitta. An inspiring collaboration for reproducibility! rOpenSci members Joel and Mo's fortuitous encounter led to an exciting targets workshop in Oslo.
+.
+
+* [My Life with the R-universe](/blog/2023/08/01/r-universe-and-cran) by Zhian N. Kamvar. Using the R-universe to rapidly deploy releases, dependencies, and bugfixes
+.
```
Comment (steffilazerte) [inline content/blog/2023-07-25-news-august-2023/index.md:141]: These two  posts have extra periods and extra spaces between periods, fix at the end?

File: content/blog/2023-07-25-news-august-2023/index.Rmd
Comment (steffilazerte) [inline content/blog/2023-07-25-news-august-2023/index.Rmd:1]: I'm never sure how much this matters, but should the folder have the date 2023-08-25?

File: content/blog/2023-07-25-news-august-2023/index.Rmd [hunk 8]
```diff
@@ -0,0 +1,489 @@
+---
+title: 'rOpenSci News Digest, August 2023'
+author: 'The rOpenSci Team'
+date: '2023-08-25'
+tags:
+  - newsletter
+slug: news-july-2023
```
Comment (steffilazerte) [inline content/blog/2023-07-25-news-august-2023/index.Rmd:7]:
```suggestion
slug: news-august-2023
```

**Review by steffilazerte — COMMENTED** [review]
Looks good! I hardly have any comments at all!

* [x] publication date is ok -> I think the folder and the slug need to be updated?
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md

File: content/blog/2023-07-25-news-august-2023/index.Rmd (hunk 6 above)
Comment (maelle) [reply content/blog/2023-07-25-news-august-2023/index.Rmd:459]: :grinning:

File: content/blog/2023-07-25-news-august-2023/index.Rmd
Comment (maelle) [reply content/blog/2023-07-25-news-august-2023/index.Rmd:1]: solved, thank you!

**Comment by maelle** [comment]
Thanks a ton @steffilazerte!

**Comment by maelle** [comment]
@yabellini thank you! Let me check that. :eyes:

**Comment by maelle** [comment]
@yabellini it was because the issue had to labels, I removed one, thanks for catching this!

**Pull request merged**